adflow/pyADflow.py

```diff
@@ -4762,6 +4782,8 @@ def _getDefaultOptions():
             "windAxis": [bool, False],
             "alphaFollowing": [bool, True],
             "TSStability": [bool, False],
+            "usetsinterpolatedgridvelocity": [bool, False],
+            "useexternaldynamicmesh": [bool, False],
```

Comment:
Can you change these to camel case so that they are easier to read?
Also, the docs build is failing because each option must be documented in the options.yaml file.

Reply:
I see. Will do that!